lib/SyTest/Homeserver.pm

```diff
@@ -241,7 +241,7 @@ sub configure_logger
 
       loggers => {
          synapse => {
-            level => "INFO"
```

Comment:
Was this just for temporary debugging?

Reply:
Oh blast, I carefully didn't add this, and then clearly did a `git commit -a`.